mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala

```diff
@@ -234,11 +235,15 @@ trait OverrideCompletions { this: MetalsGlobal =>
         if (sym.isLazy) "lazy "
         else ""
 
+      // don't show <defaultmethod>
+      val mask = sym.flagMask & ~Flags.JAVA_DEFAULTMETHOD
       val _modifs =
-        sym.flagString.replace(
-          sym.privateWithin.toString(),
-          sym.privateWithin.name.toString()
-        )
+        sym
+          .flagString(mask)
+          .replace(
+            sym.privateWithin.toString(),
+            sym.privateWithin.name.toString()
+          )
```

Comment:
There is no more need in this `replace`

Reply:
Och, how did you find that?

Reply:
It seems that the tests fail if I remove the replace since private within is doing `package a` on `toString`:
`override protected[package a] def f(): Unit = ???`

Reply:
oops, it's a different thing))
I thought that it was a prev approach to same thing and run only CompletionIssueSuite